Cargo.toml

```diff
@@ -18,8 +18,11 @@ rand_core = "0.6.4"
 serde = { version = "1", default-features = false }
 sha2 = { version = "0.10.6", default-features = false }
 subtle = { version = "2.4.1", default-features = false }
-trussed = { git = "https://github.com/trussed-dev/trussed", rev = "1c55b3b2dd6a9e1cfc55758635baf0d0bbf387d1", features = ["serde-extensions"] }
+trussed = { version = "0.1.0", features = ["serde-extensions"] }
 
 [dev-dependencies]
 rand_core = { version = "0.6.4", default-features = false, features = ["getrandom"] }
-trussed = { git = "https://github.com/trussed-dev/trussed", rev = "1c55b3b2dd6a9e1cfc55758635baf0d0bbf387d1", features = ["serde-extensions", "virt"] }
+trussed = { version = "0.1.0", features = ["serde-extensions", "virt"] }
+
+[patch.crates-io]
+trussed = { git = "https://github.com/robin-nitrokey/trussed.git", rev = "f943d88aa43c72bc76c5c8bf8c8b5bb3638a4b85" }
```

Comment:
Use https://github.com/Nitrokey/trussed/releases/tag/v0.1.0-nitrokey-5 instead for easier readability and for reproducibility?

Reply:
Actually the branch we were currently using was missing https://github.com/trussed-dev/trussed/pull/94 so I fixed it. I thought it was the other way around